scipy/interpolate/fitpack2.py

```diff
@@ -1644,8 +1738,23 @@ class RectSphereBivariateSpline(SphereBivariateSpline):
 
     See Also
     --------
-    RectBivariateSpline : bivariate spline approximation over a rectangular
-        mesh
+    BivariateSpline :
+        to create a bivariate spline on a plane grid
+    UnivariateSpline :
+        a similar class for univariate spline interpolation
+    SmoothBivariateSpline :
+        to create a bivariate spline through the given points
```

Comment:
Maybe reword? Reading this line leaves me as confused as before, and having to look up the docs. Maybe just say it's a smoothing bivariate spline and leave it at that (more details are available in the SmoothBivariareSpline docstring; the latter may use an example, but that's separate).

Reply:
I clean up the comment.